### PR TITLE
Set the console code page to "UTF-8"

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 1]
```diff
@@ -30,6 +30,10 @@ export class dotNetExe {
 
     public async execute() {
         tl.setResourcePath(path.join(__dirname, "task.json"));
+        this.setConsoleCodePage();
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: Should this actually be handled in task-lib?

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 2]
```diff
@@ -30,6 +30,10 @@ export class dotNetExe {
...
+        if (this.command === "custom") {
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: is this a merge issue?

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 3]
```diff
@@ -58,6 +62,13 @@ export class dotNetExe {
         }
     }
 
+    private  setConsoleCodePage() {
+        // set the console code page to "UTF-8"
+        if (tl.osType() === 'Windows_NT') {
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: Also, does it need to be done in other tasks where we are invoking exes like az cli, packer

File: Tasks/DotNetCoreCLI/dotnetcore.ts (hunk 2 above)
Comment (ds-ms) [reply Tasks/DotNetCoreCLI/dotnetcore.ts]: Yes, it is.

File: Tasks/DotNetCoreCLI/dotnetcore.ts (hunk 3 above)
Comment (nigurr) [reply Tasks/DotNetCoreCLI/dotnetcore.ts]: Any exception thrown from this should be ignored with a warning

**Comment by sachinma** [comment]
@ds-ms can you work on closing this

**Comment by ds-ms** [comment]
@sachinma sure

**Comment by sachinma** [comment]
ping @ds-ms

**Pull request merged**